tinacms/tinacms
# Fix/allow svg imports

## What this PR does. 

1. It adds back our ES build loaders for esbuild and adds a vite plugin that allows SVG files to be loaded as react components.
2. Fixes issue where someone would indirectly import the client from the config file by adding a dummy client
We previously handled  this   

## How to test

1. SVG imports 
- Add an SVG file to the tina folder
- import the SVG file "import SVG from './svgFile.svg'"
- Render it in `ui.component` and make sure there are no errors and it renders correctly

2. Dummy client regrestion
- Add an import to the client in the config file
- Confirm that it does not break the build
- Test CLI for regressions as I had to rearrange the code a little bit
